queries/unison/highlights.scm

```diff
@@ -12,33 +12,30 @@
 (literal_boolean) @boolean
 
 ; Keywords
+(type_kw) @keyword.type
+
 [
   (kw_forall)
-  (unique)
-  (type_kw)
   (do)
   (kw_let)
   (ability)
   (where)
 ] @keyword
 
-(kw_equals) @keyword.operator
+(kw_equals) @operator
```

Comment:
I think that should stay as it was (and similar below); operator is reserved for symbols.